### PR TITLE
chore: update read me file for post funding

### PR DESCRIPTION
**Pull request by jayanta2018**
TIS21-5812

All the relevant confluence pages have been updated.

### CODE REVIEW

**Review by JosephKelly — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ The following environmental variables must be provided
 | **PostTrainingBodyTrustJob**             |      PostTrainingBodyTrustJob       |          PostTrainingBodyTrustFullSync |                                                  none (assuming only this failed) |
 | **PersonRecordStatusJob**                |        PersonRecordStatusJob        |                  personRecordStatusJob |                        **PersonElasticSearchSyncJob** (assuming only this failed) |
 | **PersonElasticSearchSyncJob**           |       PersonElasticSearchJob        |                personElasticSearchSync |                                                                              none |
+| **PostFundingStatusSyncJob**             |      PostFundingStatusSyncJob       |         postFundingStatusSyncJob       |                                              Re-run **Post Funding Status Sync Job** |
```
[the remark this answers is not in the document]
Comment (jayanta2018) [reply README.md:29]: I beg your pardon, Pepe, what is SVG?

[the remark this answers is not in the document]
Comment (JosephKelly) [reply README.md:29]: https://github.com/Health-Education-England/TIS-SYNC/blob/main/jobDependency.svg

[the remark this answers is not in the document]
Comment (JosephKelly) [reply README.md:29]: See: https://github.com/Health-Education-England/TIS-SYNC/blob/e7842301b3a3c3a12e869e7e657662804905502b/README.md?plain=1#L16

[the remark this answers is not in the document]
Comment (jayanta2018) [reply README.md]: Added and updated now

**Review by JosephKelly — APPROVED** [review]

**Pull request merged**